### PR TITLE
5.0: Update `django.contrib.contenttypes.prefetch`

### PR DESCRIPTION
**Pull request by JaeHyuckSa**
# I have made things!
Update stubs for `django.contrib.contenttypes.prefetch` for Django 5.0.

- [x] `django.contrib.contenttypes.prefetch`
  - [x]  `django.contrib.contenttypes.prefetch.GenericPrefetch` was added

## Related issues

Refs
- https://github.com/typeddjango/django-stubs/issues/1493

Upstream PR
- https://github.com/django/django/pull/17136

### CODE REVIEW

File: django-stubs/contrib/contenttypes/prefetch.pyi [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from django.db.models import Prefetch
+from django.db.models.query import QuerySet
+
+class GenericPrefetch(Prefetch):
+    def __init__(self, lookup: str, querysets: list[QuerySet] | None = ..., to_attr: str | None = ...) -> None: ...
```
Comment (sobolevn) [inline django-stubs/contrib/contenttypes/prefetch.pyi]: Please, see https://github.com/django/django/pull/18125

Comment (saJaeHyukc) [reply django-stubs/contrib/contenttypes/prefetch.pyi:7]: I've read the corresponding PR. It's a shame that if I had thought about it a little more, I would have found a bug. Still, thank you for letting me know the good opportunity :) I'm learning again today. If Django PR merges, I'll fix the code and apply it.

Comment (saJaeHyukc) [reply django-stubs/contrib/contenttypes/prefetch.pyi:7]: I have confirmed that the code has been applied! I will fix it and commit it.

**Comment by JaeHyuckSa** [comment]
I have confirmed that the code has been applied! I will fix it and commit it.

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**